CenterForOpenScience/osf.io
# [PLAT-871][PLAT-894] Handle large Zotero folders

## Purpose

This speeds up the display of citations using Zotero and removes the copy folders button. 

## Changes

- removes Zotero copy button 
- reorganize functions so they are evaluated more efficiently 

## QA Notes

Enable zotero and load up a Zotero folder with citations (200 is the display limit so > 200 in optimal). You can just drag and drop 200 random files using the Zotero app. Then simply expand the folder to see if it works. The folder should take < 30 seconds to load and produce no JS errors. Also there should be a download button, but no copy button for folders.

## Documentation

🐞 fix no docs.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/PLAT-871
https://openscience.atlassian.net/browse/PLAT-894
